tests/std/tests/P0645R10_text_formatting_args/test.cpp

```diff
@@ -0,0 +1,226 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <assert.h>
+#include <concepts>
+#include <cstddef>
+#include <format>
+#include <memory>
+#include <string_view>
+#include <type_traits>
+#include <variant>
+
+using namespace std;
+
+#pragma warning(push)
+#pragma warning(disable : 4582) // 'uninit_vector::storage': constructor is not implicitly called
+#pragma warning(disable : 4583) // 'uninit_vector::storage': destructor is not implicitly called

```

Comment:
I don't see the type for which these warnings are being suppressed. Are the suppressions no longer necessary, or are they suppressing warnings from the product code, or... ?

Reply:
They were necessary, when I still had the union member initializer and MSVC complained about it